src/Configuration/HiveConfigRowDefinition.php

```diff
@@ -83,11 +83,9 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->booleanNode('nullable')
                             ->defaultValue(false)
                         ->end()
-                        ->booleanNode('default')
-                            ->defaultValue(false)
+                            ->scalarNode('default')
```

Comment:
Preklep, `default` key nie je boolean, ... v rows configuracii to bolo nespravne definovane.